package.json

```diff
@@ -8,12 +8,13 @@
   "scripts": {
     "preinstall": "if test \"$NODE_ENV\" = \"production\"; then rm -rf public; mkdir public; fi",
     "postinstall": "if test \"$NODE_ENV\" = \"production\"; then npm run build; fi",
-    "start": "if test \"$NODE_ENV\" = \"production\"; then npm run serve; else npm run dev; fi",
+    "start": "npm run contentCache; if test \"$NODE_ENV\" = \"production\"; then npm run serve; else npm run dev; fi",
```

Comment:
I think this might need to be moved to the `postinstall` or build commands so the file is created on Heroku. I could be mistaken, and start may work, but I have a feeling it has to be the part of the `preinstall` or `postinstall` commands.